combinato/__init__.py

```diff
@@ -5,7 +5,8 @@
 
 from __future__ import absolute_import, print_function
 from .options import options, artifact_criteria
-
+import os,sys
+sys.path.insert(0, os.getcwd())
```

Comment:
Under what circumstances is it necessary to do this?

Reply:
If you have a local_options.py file in the same folder as our data and that folder is not in the python path

Reply:
I checked this in several variants on Linux and I didn't see any situation where the path modification was actually necessary. Is this a systematic difference between Python implementations on Windows and Linux?

Reply:
Yes is a windows thing. Because I have to call combinato like:

`python C:\Path\To\Combinato\css-simple-clustering
`

The directory added to the path is `C:\Path\To\Combinato\`. More info on the best answer [here](https://stackoverflow.com/questions/25085346/why-sys-path-doesnt-contain-cwd).

Reply:
Thank you for your explanation. I found why I never needed the modification you're suggesting: because my `$PYTHONPATH` started with `:`.

So I'll gladly integrate your PR, but it would be great if you could turn in into 1 commit? The current 9 commits for 8 lines of code might be a bit confusing :-)